pkg/gofr/helpers.go

```diff
@@ -0,0 +1,25 @@
+package gofr
+
+import (
+	"strings"
+)
+
+func SplitEnv(envString, splitString string) []string {
```

Comment:
I would have passed the name of the env variable here, and call the os.GetEnv here


```suggestion
func SplitEnv(envName, splitString string) []string {

envString = os.GetEnv(envName)

// …
```

This way the code calling SplitEnv is simpler.

Reply:
@KedarisettiSreeVamsi

Reply:
Hi @ccoVeille,
I have already taken the required config details in gofr.go and then I am passing the data to SplitEnv, is it really necessary to make this change.

Reply:
No, no worries